server/jetstream_cluster_4_test.go

```diff
@@ -3060,11 +3060,6 @@ func TestJetStreamClusterKeyValueLastSeqMismatch(t *testing.T) {
 	require_NoError(t, err)
 	require_Equal(t, revision, 2)
 
-	// Now delete foo from sequence 1.
-	// This needs to be low level remove (or system level) to test the condition we want here.
-	err = js.DeleteMsg("KV_mismatch", 1)
-	require_Error(t, err)
```

Comment:
This would always be an error due to `DenyDelete` being enabled on a KV by the Go client.